compiler/onecc-docker/onecc-docker

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env bash
+''''exec "python3" "onecc-docker" "$@" #'''
+
+
+
+import sys
+import subprocess
+import json
+import os
+import re
+import requests
+
+
+def _is_image(version) : 
+    cmd = ['docker', 'images', f'onecc:{version}']
+    p = subprocess.Popen(cmd,  stdout=subprocess.PIPE, stderr=subprocess.STDOUT) 
+    out, err = p.communicate()
+    dic = {}
+
+    decoded_out = [i.split() for i in out.decode().split("\n") if len(i)]
+
+    if len(decoded_out) <2 :
+        return False
+
+    dic[decoded_out[0][0]] = decoded_out[1][0]
+    dic[decoded_out[0][1]] = decoded_out[1][1]
+
+    tag_name = cmd[2].split(':')
+
+    if dic['REPOSITORY'] == tag_name[0] and dic['TAG'] == tag_name[1]  :
+        return True 
+    else :
+        return False
+
+def write_dockerfile(script_path,version) :
+
+    with open(f'{script_path}/Dockerfile', 'w', encoding='utf-8') as dockerfile: 
+        dockerfile.write('FROM ubuntu:18.04 \n\n')
+        dockerfile.write('RUN apt update -y && \ \n')
+        dockerfile.write('\t apt install -y \ \n')
+        dockerfile.write('\t wget && \ \n')
+        dockerfile.write('\t apt clean \n')
+        dockerfile.write(f'RUN wget https://github.com/Samsung/ONE/releases/download/{version}/one-compiler_{version}_amd64.deb && \ \n')
+        dockerfile.write(f'\t apt-get install -y ./one-compiler_{version}_amd64.deb \n')
+        dockerfile.write('ENTRYPOINT ["onecc"]')
```

Comment:
Currently, we check the latest version of "ONE" with github api and dynamically create a Dockerfile. Is this all right? Or is it better to commit one Dockerfile statically and use it continuously?

Reply:
This way is good to me. 

Another way I thought was to create static Dockerfile with declaring version as a variable, and update the variable in bash command.

I think both ways are worth trying.

However, what about using the static Dockerfile with the fixed version like you implemented before?

Reply:
You can just parameterize the version with `--build-arg` option. Please see https://docs.docker.com/engine/reference/commandline/build/#set-build-time-variables---build-arg for more detail.